lib/voice/Piper.js

```diff
@@ -105,7 +105,8 @@ class Piper extends EventEmitter {
             this.streams.push(source.once("error", (e) => this.stop(e)));
         }
 
-        if(options.format === "opusPackets") { // eslint-disable no-empty
+        // eslint-disable-next-line no-empty
+        if(options.format === "opusPackets") {
```

Comment:
Can probably remove this if it isn't being used? It isn't in the options list nor is it referenced anywhere else

Reply:
Not outdated, still relevant https://github.com/abalabahaha/eris/pull/1535/files#diff-19af45059193a814c9afad142eedc5c6f414dc6b0a0de15493cef7524d9da874R108-R109

Reply:
would be out of scope with this PR, but idk if @abalabahaha wants to change it here or leave it for another PR